code/modules/surgery/autopsy.dm

```diff
@@ -26,24 +30,50 @@
 	var frNum = ++autopsyPrintCount
 
 	N.name = text("AR-[] 'Autopsy Record'",frNum)
-	N.info = text("<center><B>Autopsy Record - (FR-[])</B></center><HR><BR>",frNum)
+	N.info = text("<center><B>Autopsy Record - (AR-[])</B></center><HR><BR>",frNum)
 	N.info += "<HR><B>Notes:</B><BR>"
 
-	for(var/O in target.damage_record)
+	for(var/datum/autopsy_record/AR in target.damage_record)
 
-		if(istype(O,/obj/item))
-			var/obj/item/I = O
-			N.info += "Analysis shows that [target] was attacked by a  [I.sharpness == IS_BLUNT ? "blunt" : "sharp"] object [I.autopsy_description != null ? "dealing [I.autopsy_description]" : "" ] causing [I.force > 10 ? "[I.force > 20 ? "devastating" : "deep" ]" : "shallow" ]  [I.damtype] damage.<BR><BR>" //double breaks for easier reading
+		if(AR.wound_type == WEAPON_WOUND)
+			N.info += "Analysis shows that [target] was attacked by a  [AR.sharpness == IS_BLUNT ? "blunt" : "sharp"] object [AR.special_description != null ? "dealing [AR.special_description]" : "" ] causing [AR.damage > 10 ? "[AR.damage > 20 ? "devastating" : "deep" ]" : "shallow" ]  [AR.damage_type] damage.<BR><BR>" //double breaks for easier reading
```

Comment:
considering the word is actually damage and not wounds, i'd rephrase the tiers into minor damage, major damage, and devastating damage